lib/reek/code_comment.rb

```diff
@@ -20,19 +21,42 @@ class CodeComment
     DISABLE_DETECTOR_CONFIGURATION = '{ enabled: false }'.freeze
     MINIMUM_CONTENT_LENGTH         = 2
     LEGACY_SEPARATOR               = ':'.freeze
+    UNKNOWN_SMELL_DETECTOR_MESSAGE = <<-EOS.freeze
```

Comment:
If you wrap `EOS` in single quotes then it will disable string interpolation FYI (equivalent to a string with single quotes)